src/schema/profile.ts

```diff
@@ -0,0 +1,14 @@
+import { z } from 'zod';
+
+export const ProfileSchema = z
+  .object({
+    email: z.string().email(),
+    password: z.string().optional(),
+    currentPassword: z.string().optional(),
+    newPassword: z.string().optional(),
+    confirmPassword: z.string().optional(),
+  })
+  .refine((data) => data.newPassword === data.confirmPassword, {
+    message: "Passwords don't match",
```

Comment:
je me demande si tu peux pas aussi gérer la trad pour ce message "Passwords don't match". Not sure, mais si possible ça serait cool